foreman/foreman.spec

```diff
@@ -301,6 +306,68 @@ BuildRequires: %{?scl_prefix}rubygem(facter)
 BuildRequires: facter
 %endif
 
+%description core
+Meta Package to install foreman core source code with minimal dependencies
+
+%files core
+%defattr(-,root,root,0755)
+%doc CHANGELOG Contributors LICENSE README.md VERSION
+%dir %{_datadir}/%{name}
+%{_datadir}/%{name}/app
+%exclude %{_datadir}/%{name}/app/assets
+%exclude %{_datadir}/%{name}/script/%{name}-debug.d
+%dir %{_datadir}/%{name}/bundler.d
+%exclude %{_datadir}/%{name}/bundler.d/development.rb
+%exclude %{_datadir}/%{name}/bundler.d/foreman_tasks.rb
```

Comment:
I meant more the fact, that the spec for foreman-tasks would need to look differently for for <= 1.14 releases, unless I miss something?